src/api/entitlements.js

```diff
@@ -263,4 +265,17 @@ export class Entitlement {
       : [json];
     return jsonList.map(json => Entitlement.parseFromJson(json));
   }
+
+  /**
+   * Returns the SKU associated with this entitlement.
+   * @return {?string}
+   */
+  getSku() {
+    return (
+      /** @type {?string} */ (getPropertyFromJsonString(
```

Comment:
If getPropertyFromJsonString returns {?string} you won't need this cast

Reply:
In this case it returns a string but that is not generically guaranteed for any random object passed in.